.travis-build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# This script compiles Guava, performing type-checking.
```

Comment:
Before, we were using the typetest/guava-typecheck job to run the type checkers on Guava.  Are we still going to use that?

Reply:
Yes, but this permits tests on branches which that does not.